ledger/ledger-utxo-observable/src/main/kotlin/net/corda/v5/ledger/utxo/observer/UtxoToken.kt

```diff
@@ -0,0 +1,18 @@
+package net.corda.v5.ledger.utxo.observer
+
+import net.corda.v5.ledger.utxo.token.selection.TokenClaimCriteria
+import net.corda.v5.ledger.utxo.token.selection.TokenSelection
+import java.math.BigDecimal
+
+/**
+ * The [UtxoToken] defines the structure of a token produced by an instance of [UtxoLedgerTokenStateObserver]
+ *
+ * @property poolKey The key of the token pool this token belongs to.
```

Comment:
Need to state explicitly that all tokens in a pool are regarded as 'fungible' and can be spent together, merged, etc

Reply:
As discussed, will follow up with the UX team about where best to add higher level documentation.